ush/wave_prnc_ice.sh

```diff
@@ -40,31 +38,25 @@
 #     The tested variables should be exported by the postprocessor script.
 
   set +x
-  echo ' '
-  echo '+--------------------------------+'
-  echo '!         Make ice fields        |'
-  echo '+--------------------------------+'
-  echo "   Model TAG       : $WAV_MOD_TAG"
-  echo "   Model ID        : ${RUN}.wave"
-  echo "   Ice grid ID     : $WAVEICE_FID"
-  echo "   Ice file        : $WAVICEFILE"
-  echo ' '
-  set_trace
-  echo "Making ice fields."
+  cat << EOF
+
++--------------------------------+
+!         Make ice fields        |
++--------------------------------+
+   Model TAG       : $WAV_MOD_TAG
+   Model ID        : ${RUN}.wave
+   Ice grid ID     : $WAVEICE_FID
+   Ice file        : $WAVICEFILE
+
+Making ice fields.
+EOF
 
   if [[ -z "${YMDH}" ]] || [[ -z "${cycle}" ]] || \
      [[ -z "${COMOUT_WAVE_PREP}" ]] || [[ -z "${FIXgfs}" ]] || [[ -z "${EXECgfs}" ]] || \
      [[ -z "${WAV_MOD_TAG}" ]] || [[ -z "${WAVEICE_FID}" ]] || [[ -z "${COMIN_OBS}" ]]; then
 
-    set +x
-    echo ' '
-    echo '**************************************************'
-    echo '*** EXPORTED VARIABLES IN preprocessor NOT SET ***'
-    echo '**************************************************'
-    echo ' '
+    echo 'ERROR: EXPORTED VARIABLES IN preprocessor NOT SET ***'
```

Comment:
Should this be a WARNING or FATAL ERROR?

Reply:
Should be an error.  NCO wants to see a FATAL ERROR message when the job fails.  This message is a precursor to that happening -- it will exit with a non-zero status, which will then be caught by the wave init ex-script.

Reply:
Got it, thanks!